db/seed.sql

```diff
@@ -4069,6 +4071,7 @@ INSERT INTO `cms_page` VALUES (5,'Copyright','1column',NULL,NULL,'copyright','Co
nbsp; No repeats or re runs are authorised.&nbsp; A new application is necessary.</p>\r\n<p>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The negatives or prints of artworks must be reproduced without cropping, bleeding, alteration or overprint of any kind.&nbsp; A colour proof must be submitted to the Administrative Officer prior to colour reproduction on book jackets, posters, cards and calendars.&nbsp; A colour proof may be requested for general illustrations in some instances.</p>\r\n<p>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The title of the work as supplied by the Society must be used at all times and acknowledgement of the Royal Western Australian Historical Society as the owner must be included with all reproductions.</p>\r\n<p>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The name of the artist, the title, accession number and credit to the RWAHS must appear in the publication.</p>\r\n<p>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Only under exceptional circumstances will an outside photographer contracted by the client be permitted access to the collection.&nbsp; In the latter case it is strictly a condition that all film stock used remains the Society&rsquo;s property and is to be returned after first publication.&nbsp; All copies of artworks must be delivered to the Society by the photographer prior to dispatch to client, for checking by Society staff.</p>\r\n<p>8.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The Society&rsquo;s 35mm or 70mm reference copy of transparencies of artworks will be supplied for reproduction purposes.&nbsp; Master copy will not be used for this purpose.</p>\r\n<p>9.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Copyright is the responsibility of the client.&nbsp; If the Society does not hold copyright ownership, the client will be required to provide a licence or written permission to reproduce from the copyright holder, prior to the supply of photographic material.&nbsp; The client will be required to sign an indemnification as part of their order form.</p>\r\n<p>10.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The type of publication must be clearly indicated:&nbsp; eg general book, artbook illustration, CD cover, text book, brochure, calendar, poster, exhibition, exhibition catalogue, reproduction print, CD rom, television, web page.</p>\r\n<p>11.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Photographs must not be photocopied, as this process damages photographs by exposing them to heat, excessive light and ozone.</p>\r\n<p>12.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The Society reserves the right to refuse permission for reproduction of a photograph, artwork or other printed item if it considers that this is not in the best interests of the Society.</p>','2016-09-12 13:31:06','2016-09-12 13:37:29',1,0,NULL,NULL,NULL,NULL,NULL,NULL);
```

Comment:
Copied and pasted from page on production website. It's common with legal stuff to use preformatted text, not sure why, portability?